src/api/Flooq.Api/Flooq.Api.csproj

```diff
@@ -17,10 +17,8 @@
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="6.0.3" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
   </ItemGroup>
+  
   <ItemGroup>
-    <SonarQubeSetting Include="sonar.exclusions">
-      <Value>Flooq.Api/Program.cs</Value>
-      <Value>**/Migrations/**</Value>
-    </SonarQubeSetting>
```

Comment:
Why are these exclusions dropped?

Reply:
??? I have changed this file, but can't remember changing anything on the Sonar configuration. Shall I remove this?

Reply:
Yeah these Sonar settings are needed to exclude the Program.cs and Migrations from the code coverage.